lib/src/query_service.dart

```diff
@@ -17,8 +18,9 @@ class QueryService {
         path: _generateQuery(advertiserId, insertionOrderId), method: 'GET');
 
     await GoogleAPI.client.request(requestArgs).then(allowInterop((response) {
-      final rawResponse = stringify(response.result);
-      _insertionOrderLists.add(InsertionOrderParser.parse(rawResponse));
+      _insertionOrderLists
+          .add(InsertionOrderParser.parse(stringify(response.result)));
+      ExcelDart.populate(_insertionOrderLists);
```

Comment:
i dont think service should populate. whatever called the service should populate and parse. this uncouples the service from the excel piece.